sklearn/decomposition/kernel_pca.py

```diff
@@ -81,6 +81,11 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         A pseudo random number generator used for the initialization of the
         residuals when eigen_solver == 'arpack'.
 
+    copy : boolean, default=True
```

Comment:
Though, at the same time, explicit > implicit, and here we have the freedom to name it what we want without a deprecation cycle.